image/cli/mascli/functions/pipeline_config_db2

```diff
@@ -193,8 +193,8 @@ function config_pipeline_storage_db2() {
                 export LOCAL_MAS_CONFIG_DIR_ALREADY_CHOSEN=yes
               fi
               jdbc_cfg_file=$LOCAL_MAS_CONFIG_DIR/jdbc-$MAS_INSTANCE_ID-wsapp.yaml
-              python3 $DIR/functions/gencfg jdbc $MAS_INSTANCE_ID $MAS_WORKSPACE_ID $MAS_APP_ID workspace-application $jdbc_cfg_file --workspace-id $MAS_WORKSPACE_ID --application-id manage
-              DB2_ACTION_MANAGE=byo
+              python3 $DIR/functions/gencfg jdbc $MAS_INSTANCE_ID $MAS_WORKSPACE_ID $MAS_APP_ID workspace-application $jdbc_cfg_file --workspace-id $MAS_WORKSPACE_ID --application-id manage           
+              DB2_ACTION_SYSTEM=byo
```

Comment:
Why change this?  We're in the code path that is setting up the Manage Db2, so we need to set `DB2_ACTION_MANAGE` as it currently does; unless I'm missing something here!?

Reply:
fixed